src/spec-configuration/containerFeaturesOCI.ts

```diff
@@ -255,13 +255,13 @@ export async function fetchRegistryAuthToken(output: Log, registry: string, reso
 
     const authReq = await request(options, output);
     if (!authReq) {
-        output.write('Failed to get registry auth token', LogLevel.Error);
+        output.write('(!) ERR: Failed to get registry auth token', LogLevel.Error);
```

Comment:
These lines will likely cause a merge conflict when I merge #99. I'd suggest removing any changes to this file